src/components/DiffView/index.spec.tsx

```diff
@@ -597,8 +597,7 @@ describe(__filename, () => {
     const root = render({ version });
 
     const provider = root.find(LinterProvider);
-    expect(provider).toHaveProp('versionId', version.id);
-    expect(provider).toHaveProp('validationURL', version.validationURL);
```

Comment:
Do we have an issue in addons-server to remove this property from the API response?

Reply:
We don't. I wasn't sure if we should do that, but I suppose we're the only consumers right now, so it shouldn't affect anyone. @diox Do you think we're fine to remove this property from the API response as Code Manager no longer needs it?